app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java

```diff
@@ -1078,7 +1078,6 @@ private void updateActionBarTitle(int page) {
     String sura = QuranInfo.getSuraNameFromPage(this, page, true);
     ActionBar actionBar = getSupportActionBar();
     if (actionBar != null) {
-      translationsSpinner.setVisibility(View.GONE);
```

Comment:
The problem is that `updateActionBarSpinner()` is calling `updateActionBarTitle()` . 
I tried to figure out the reason for hiding translation spinner, but I didn't notice the reason.
May be @ahmedre has any thoughts?